vispy/color/colormap.py

```diff
@@ -403,7 +451,8 @@ class Colormap(BaseColormap):
 
     """
 
-    def __init__(self, colors, controls=None, interpolation='linear'):
+    def __init__(self, colors, controls=None, interpolation='linear',
```

Comment:
super-nit: consider making the new parameters kw-only for clarity and future proofing.